src/commands/Minion/slayertask.ts

```diff
@@ -32,13 +34,42 @@ export default class extends BotCommand {
 		});
 	}
 
+	public getAlternateMonsterList(currentTask: SlayerTaskTable, assignedTask: AssignableSlayerTask | null) {
+		if (currentTask && assignedTask) {
+			const altMobs = assignedTask.monsters;
+			const alternateMonsters = killableMonsters
+				.filter(m => {
+					return altMobs.includes(m.id) && m!.id !== assignedTask.monster.id;
+				})
+				.map(m => {
+					return m!.name;
+				});
+			return alternateMonsters.length > 0 ? ` (${alternateMonsters.join('/')})` : '';
+		}
+		return '';
+	}
+
+	public async returnSuccess(msg: KlasaMessage, message: string, autoslay: boolean) {
+		if (autoslay) {
+			// await msg.channel.send(message);
```

Comment:
Remove `// await` so the task assignment message comes through without problems. Magna might change this later, but I think people will want to see the message.

Reply:
The purpose of --as is to ignore all that and just train slayer. Having the first message will not be helpful for most people and will just create stress on the bot as multiple messages will be sent for little to no reasons. I left it there so it can be re-toggled as easily, if wanted.